angr/analyses/calling_convention.py

```diff
@@ -142,4 +142,23 @@ def _is_sane_register_variable(self, variable):
         return True
 
 
+def recover_calling_conventions(project, kb=None):
```

Comment:
Nitpick: I prefer putting it as a static method under the corresponding class (in this case, the `CallingConventionAnalysis` class). What do you think?

Reply:
I choose this way, because its not a utility method _used by_ the other methods of the class, but instead a utility function which helps the usage of the class itself. But I don't see any problems of having this as a static method of the class, especially if it fits with the current project coding conventions. Will fix it and re-push the PR.

Reply:
Personally I chose to follow a stricter OO design :-)

Thanks for (agreeing on) fixing it!